### DISCUSSION TITLE
Classic stabilization?

### DISCUSSION THREAD

**Discussion by scribblepleb** [keiwando/toonsquid#319] [Ideas]
Hi! I’m not sure how easy this would be to implement, but if possible, I think it might be nice to have an option for more traditional stabilization instead of just smoothing for your brushes. In all the drawing apps I’ve used, I’ve grown accustomed to a certain type of stabilization, and I’m sure other artists feel this way, as well. Toonsquid’s smoothing strays a bit from the norm, as I’ve noticed, and it can be a little offputting while I draw. So I think it might be helpful to have multiple stabilization options! Or, maybe, just more customization options when it comes to stabilization, like for pressure or speed.
Thank you!

**Comment by keiwando** [comment]
Thanks for the feedback! For my reference, would you mind sharing one or two programs that you're thinking of that have that certain kind of stabilization behaviour?

**Reply by DinoAwakening** [reply]
RoughAnimator-like stabilization would be great :0

**Reply by scribblepleb** [reply]
Something maybe similar to Procreate’s stabilization option, or maybe even Clip Studio’s would be awesome!
